Mathlib/Data/NNRat/BigOperators.lean

```diff
@@ -14,38 +14,54 @@ variable {α : Type*}
 
 namespace NNRat
 
+section DivisionSemiring
+
+variable {K : Type} [DivisionSemiring K] [CharZero K]
+
+@[norm_cast]
+theorem coe_list_sum (l : List ℚ≥0) : (l.sum : K) = (l.map (↑)).sum :=
```

Comment:
```suggestion
theorem cast_listSum (l : List ℚ≥0) : (l.sum : K) = (l.map (↑)).sum :=
```
since `coe` is reserved for the `ℚ≥0 → ℚ` coercion, and the more general one is called `Rat.cast`. Same below. You will need to deprecate the existing lemmas.